featuretools/entityset/entityset.py

```diff
@@ -1776,6 +1784,29 @@ def _add_references_to_metadata(self, dataframe):
             metadata.update(entityset_id=self.id)
         _ES_REF[self.id] = self
 
+    def _normalize_dataframe(self, dataframe):
```

Comment:
`_normalize_dataframe` might not be the best term to use since `EntitySet.normalize_dataframe` already exists and is used to split one dataframe into two dataframes